static-site/src/pages/team.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "styled-components";
+import GenericPage from "../layouts/generic-page";
+import { TeamMember, BigSpacer, FlexCenter} from "../layouts/generalComponents";
+import { CenteredFocusParagraph } from "../components/splash/styles";
+
+/**
+ * Todo - we would like a brief sentence or two describing what people work on
+ * (or worked on) here. We need to both write the sentence and then render this
+ * (only in the grid, not in the footer).
+ */
+
+export const teamMembers = [
+  {name: "Trevor Bedford", image: "trevor-bedford.jpg", link: "http://bedford.io/team/trevor-bedford/"},
+  {name: "Richard Neher", image: "richard-neher.jpg", link: "https://neherlab.org/richard-neher.html"},
+  {name: "James Hadfield", image: "james-hadfield.jpg", link: "http://bedford.io/team/james-hadfield/"},
+  {name: "Emma Hodcroft", image: "emma-hodcroft.jpg", link: "http://emmahodcroft.com/"},
+  {name: "Thomas Sibley", image: "thomas-sibley.jpg", link: "https://bedford.io/team/thomas-sibley/"},
+  {name: "John Huddleston", image: "john-huddleston.jpg", link: "http://bedford.io/team/john-huddleston/"},
+  {name: "Ivan Aksamentov", image: "ivan-aksamentov.jpg", link: "https://neherlab.org/ivan-aksamentov.html"},
+  {name: "Jover Lee", image: "jover-lee.jpg", link: "http://bedford.io/team/jover-lee/"},
+  {name: "Jennifer Chang", image: "jennifer-chang.jpg", link: "https://bedford.io/team/jennifer-chang/"},
+  {name: "Victor Lin", image: "victor-lin.png", link: "https://bedford.io/team/victor-lin/"},
+  {name: "Cornelius Roemer", image: "cornelius-roemer.jpg", link: "https://neherlab.org/cornelius-roemer.html"},
+];
+
+const alumni = [
+  {name: "Cassia Wagner", image: "cassia-wagner.jpg", link: "https://bedford.io/team/cassia-wagner/"},
+  {name: "Miguel Parades", image: "miguel-parades.jpg", link: "https://bedford.io/team/miguel-parades/"},
```

Comment:
It was at first a little weird to see active Blab members listed as alumni, but I think I understand why: they were actively contributing to SARS-CoV-2 analyses and narratives a few years ago, but no longer are, right?

Reply:
Yeah - I talked about this with Trevor & the rough thinking was core = those who are at the biweekly meetings. But happy either way - I agree it's a little strange!